Patches/Android Tiers/Hediffs/Hediffs_Android.xml

```diff
@@ -145,7 +145,7 @@
 					</capacities>
 					<power>44</power>
 					<cooldownTime>2.51</cooldownTime>
-					<armorPenetrationSharp>14.07</armorPenetrationSharp>
+					<armorPenetrationSharp>9.38</armorPenetrationSharp>
 					<armorPenetrationBlunt>45</armorPenetrationBlunt>

```

Comment:
The new sharp value is acceptable. The blunt AP seems high for a bite, but I presume it's to get through power armor?

Reply:
It is pretty much strictly to let it get through power armor, yeah.